updates/5/updates.sql

```diff
@@ -0,0 +1,52 @@
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("ROSN", "S", "Roseon Finance", "ROSN", 1620551645, NULL, "roseon-finance", "", "ROSN"); INSERT INTO common_asset_details(asset_id, forked) VALUES("ROSN", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xC52C326331E9Ce41F04484d3B5E5648158028804", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xC52C326331E9Ce41F04484d3B5E5648158028804", "C", "ZEN Exchange Token", "ZCX", 1607683983, NULL, "unizen", "ZCX", "0xC52C326331E9Ce41F04484d3B5E5648158028804");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x755be920943eA95e39eE2DC437b268917B580D6e", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x755be920943eA95e39eE2DC437b268917B580D6e", "C", "VersoView", "VVT", 1605013606, NULL, "versoview", "", "0x755be920943eA95e39eE2DC437b268917B580D6e");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xA2120b9e674d3fC3875f415A7DF52e382F141225", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xA2120b9e674d3fC3875f415A7DF52e382F141225", "C", "Automata", "ATA", 1622409616, NULL, "automata", "ATA", "0xA2120b9e674d3fC3875f415A7DF52e382F141225");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x21381e026Ad6d8266244f2A583b35F9E4413FA2a", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x21381e026Ad6d8266244f2A583b35F9E4413FA2a", "C", "Formation Finance", "FORM", 1617301388, NULL, "formation-fi", "", "0x21381e026Ad6d8266244f2A583b35F9E4413FA2a");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xf4CD3d3Fda8d7Fd6C5a500203e38640A70Bf9577", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xf4CD3d3Fda8d7Fd6C5a500203e38640A70Bf9577", "C", "YfDAI.finance", "Yf-DAI", 1599024013, NULL, "yfdai-finance", "YFDAI", "0xf4CD3d3Fda8d7Fd6C5a500203e38640A70Bf9577");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("CQT", "X", "Covalent", "CQT", 1614214958, NULL, "covalent", "", "CQT"); INSERT INTO common_asset_details(asset_id, forked) VALUES("CQT", NULL);
```

Comment:
Why do you have this marked as an avalanche token? From looking around it seems like it's not launched yet. So why add a non launched token?